installation/resources/installer-cr-cluster.yaml.tpl

```
@@ -32,13 +32,8 @@ spec:
       namespace: "knative-serving"
     - name: "knative-eventing"
       namespace: "knative-eventing"
-   # - name: "knative-eventing-channel-kafka"
-   #   namespace: knative-eventing
-   #   source:
-   #     # NOTE: this is a go-getter compatible url
-   #     url: github.com/kyma-incubator/knative-kafka.git//resources/knative-kafka?ref=master
-   # - name: "knative-eventing-channel-kafka-tests"
-   #   namespace: knative-eventing
+    - name: "knative-eventing-kafka"
```

knative-eventing-kafka is an optional component because it requires a Kafka cluster and installation overrides.

```suggestion
    # - name: "knative-eventing-kafka"
    #   namespace: knative-eventing    
```

It should be changed in [installation/resources/installer-cr-gardener-azure.yaml.tpl](https://github.com/kyma-project/kyma/blob/deafba3c8e4a8d7502d07074fd39ac54d4cab44d/installation/resources/installer-cr-gardener-azure.yaml.tpl#L39) since the `kyma-gardener-azure-integration` prow job comes with kafka cluster and the requires installation overrides